ct-app/test/components/test_conversions.py

```
@@ -0,0 +1,9 @@
+from core.components.balance import Balance
+from core.components.conversions import convert_unit
+
+
+def test_convertUnit():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)